2.train-VAE/1.optimize_hyperparameters.py

```diff
@@ -76,6 +76,6 @@
 tuner.search(
         train_df,
         validation_data=(test_df, None),
-        callbacks=[EarlyStopping("val_loss", patience=10)],
+        #callbacks=[EarlyStopping("val_loss", patience=10)],
```

Comment:
Can you add a comment about why we didn't use this? It will be good for us to remember in the future!